dutchx_scripts/daoCreateDutchX.ts

```diff
@@ -0,0 +1,227 @@
+  const auction4Reputation =
+    await Auction4ReputationFactory.at((await contractNew(web3, networkName, { name: "Auction4Reputation" }, "max", "50000000000") as Lock4ReputationContract).address);
+
+  let externalTokenLockerAddress: Address;
+  let priceOracleInterfaceAddress: Address;
+  let priceOracleInterfaceMock: any;
+  let gnoTokenAddress: Address;
+  let externalTokenLockerMock: { lock: (amount: string, options: { from: Address }) => Promise<void>, address: Address };
+  const genTokenAddress = await Utils.getGenTokenAddress();
+
+  switch (networkName) {
+    case "Ganache":
+      gnoTokenAddress = genTokenAddress;
+
+      priceOracleInterfaceMock =
+        await contractNew(web3, networkName, { name: "PriceOracleMock" }, "max", "50000000000") as any;
+
+      priceOracleInterfaceAddress = priceOracleInterfaceMock.address;
+
+      await priceOracleInterfaceMock.setTokenPrice(gnoTokenAddress, 380407, 200000000);
+
+      externalTokenLockerMock =
+        await contractNew(web3, networkName, { name: "ExternalTokenLockerMock" }, "max", "50000000000") as any;
+
+      externalTokenLockerAddress = externalTokenLockerMock.address;
+
+      await externalTokenLockerMock.lock("100000000000000000000", { from: accounts[0] });
+      await externalTokenLockerMock.lock("100000000000000000000", { from: accounts[1] });
+      await externalTokenLockerMock.lock("100000000000000000000", { from: accounts[2] });
+
+      break;
+    case "Kovan":
+      externalTokenLockerAddress = "0x4edc383adea781762b74e7082c03f423523e61bb";
+      gnoTokenAddress = "0x6018bf616ec9db02f90c8c8529ddadc10a5c29dc";
+
+      priceOracleInterfaceMock =
+        await contractNew(web3, networkName, { name: "PriceOracleMock" }, "max", "50000000000") as any;
+
+      priceOracleInterfaceAddress = priceOracleInterfaceMock.address;
+
+      await priceOracleInterfaceMock.setTokenPrice(gnoTokenAddress, 380407, 200000000);
+      await priceOracleInterfaceMock.setTokenPrice("0x4edc383adea781762b74e7082c03f423523e61bb", 380407, 200000000);
+      await priceOracleInterfaceMock.setTokenPrice("0x543Ff227F64Aa17eA132Bf9886cAb5DB55DCAddf", 380407, 200000000);
+
+      externalTokenLockerMock =
+        await contractNew(web3, networkName, { name: "ExternalTokenLockerMock" }, "max", "50000000000") as any;
+
+      externalTokenLockerAddress = externalTokenLockerMock.address;
+
+      await externalTokenLockerMock.lock("100000000000000000000", { from: accounts[0] });
+
+      break;
+    case "Rinkeby":
+      externalTokenLockerAddress = "0x4edc383adea781762b74e7082c03f423523e61bb";
+      gnoTokenAddress = "0xd0dab4e640d95e9e8a47545598c33e31bdb53c7c";
+      /**
+       * This is just one they recently deployed.  It changes with each new DAO.  getPrice currently returns all zeros, need to find a better one
+       */
+      priceOracleInterfaceAddress = "0x1C54f6146bA3656739A5c0781Cc054FA0C3951C1";
```

Comment:
what is this ?

Reply:
it is gone  https://github.com/daostack/dx-deployment-script/pull/1/commits/a376b5a0bc59d8546654d35d67903874741d8db3